cloudtrail-lambda-dynamo-cdk/README.md

```diff
@@ -29,6 +28,10 @@ s.
     ```
     npm install
     ```
+1. Open the lambda function cloudtrail-lambda-dynamo-cdk/src/lib/lambda/object_tag_checker/index.py, and on line 17 replace the example keys with your required S3 object keys:
```

Comment:
should be `resource_tag_checker` folder for Lambda. and Line 40

Reply:
Suggest also wording this as an optional for your real life use-case. For the test, you don't need to modify these. If I understand correctly it's not S3 object keys, its the S3 object key *tags* that are being checked.  So, speficy that you can amend the tags associated with the S3 objects that you want to check